core/src/test/java/org/apache/accumulo/core/metadata/schema/TabletFileTest.java

```diff
@@ -40,6 +41,16 @@ private TabletFile test(String metadataEntry, String volume, String tableId, Str
     return tabletFile;
   }
 
+  @Test
+  public void testEquivalence() {
```

Comment:
Could have a static function in DataFileManager that does the conversion and then have a unit test calls that function and verify the result.

Reply:
Created a utility function for this and moved the test to a better location in 9d1373b